inc/functions_user.php

```diff
@@ -725,9 +725,12 @@ function get_pm_folder_name($fid, $name="")
 
 	switch($fid)
 	{
-		case 1:
```

Comment:
As these numbers are used in multiple places, I would be tempted to `define()` some constants for them to prevent possible future bugs.

Reply:
Are you sure? `$fid` has been used as is in multiple locations of templates.
A constant in template?